### PR TITLE
[#10299] Fix some misaligned features compared to V6

### PR DESCRIPTION
**Pull request by wkurniawan07**
Part of #10299 

Removed the following:
- Instructor course details page: "Non-English character not showing" modal
- Instructor course enroll page: "Hide enroll result"

Fixed the following:
- Instructor session results page: "Print view" should not expand all tabs
  - Also, hide the moderate response and add comment buttons in print view, because they are obviously useless and wasting space there
- Instructor add new course: no need for further instruction in the toast message; just stop the message at "the course has been added".
- Instructor add new course: display timezone offset in form (similar to edit course; it was missed before because I didn't realize they're not the same component)
- Student course details (view team) page: revert look-and-feel and information organization to V6 style
  - New features (sorting, more parts of student profile shown) are retained
  - Unnecessary information (course created at, course time zone) are removed
  Screen Shot 2020-07-15 at 11 09 44 PM
- Instructor session results page: Contribution question (empty) statistics table is shown when no responses have been submitted
- Instructor add comment: Edit visibility of comment for existing comment is missing

### CODE REVIEW

**Comment by wkurniawan07** [comment]
> the Sort By buttons should be changed to display on the right side, like that of the instructor home page

Wouldn't that be weird? Instructor home page has a table that spans the entire page width, but here the team members' profiles only span barely half the page (or more depending on the content, but you get the idea).

**Comment by t-cheepeng** [comment]
> but here the team members' profiles only span barely half the page

Okay that makes sense. Though I think it is in the spirit of standardizing the look/feel of the pages, The sort by buttons in this page feels like a stark difference from the other pages, maybe due to the grey background on it?

File: src/web/app/pages-instructor/instructor-course-enroll-page/instructor-course-enroll-page.component.ts [hunk 1]
```diff
@@ -341,15 +341,6 @@ export class InstructorCourseEnrollPageComponent implements OnInit {
     this.ngbModal.open(pasteModalBox);
   }
 
-  /**
-   * Reset page to default view
-   */
-  hideEnrollResults(): void {
-    this.showEnrollResults = false;
```
Comment (xpdavid) [inline src/web/app/pages-instructor/instructor-course-enroll-page/instructor-course-enroll-page.component.ts:348]: So we can also remove the variable?

Comment (wkurniawan07) [reply src/web/app/pages-instructor/instructor-course-enroll-page/instructor-course-enroll-page.component.ts:348]: Not really. This is only shown after enrollment, not all the time in the page.

**Pull request merged**